### DISCUSSION TITLE
Dynamically register a provider on Adonis V5

### DISCUSSION THREAD

**Discussion by Xstoudi** [adonisjs/core#1548] [Help] [answered]
Hello,

In Adonis V4, we could register service providers dynamically as we used to edit the `start/app.js` to build the `providers` list dynamically.

In Adonis V5, providers are registered from the `.adonisrc.json` and we cannot do such thing.

Is there an other way to do that from the ignitor/bootstrapper API by editing `server.ts`?

Thank you very much.

**Comment by thetutlage** [comment]
What's the use case?

**Comment by Xstoudi** [comment, marked as answer]
I finally created a script that edit the .adonisrc.json according to what was discussed in the RFC ^^
